### DISCUSSION TITLE
how to show reviews from facebook on site?

### DISCUSSION THREAD

**Discussion by babybeddingdesign** [swellstores/community#60] [Help] [answered]
how i can show my reviews from my facebook business page on my site?

**Comment by greph** [comment, marked as answer]
Hi @babybeddingdesign , the Product model currently supports reviews (with ratings). Though not visible in the product dashboard UI, those fields can be referenced in your Model Explorer (Developer -> Model Explorer) by searching for product reviews.

Alternatively, a custom "Review" model could be created, and those fields linked to the product model if you'd like to see them in the UI.

**Reply by babybeddingdesign** [reply]
i am not a dev and i don't understand all of these codes. i am looking for a way to show my facebook feedback on my site (not per product) on the homepage. can this be done?

i am playing with swell and thinking of moving my Magento site to it. the idea is I like to get a very fast site that has good SEO. i am also testing strapi.io .
Does anyone have moved from magneto to one of these new platforms?

**Reply by ericingram** [reply]
It's going to take custom dev work at the moment
